rdkcentral/Dobby
# [RDK-53519] added mount tunnel based addMount() implementation

### Description
add support for dynamic mounting of devices/dirs inside containers on devices with older linux kernels (<5.4)


### Test Procedure
Container config should have `rootfsPropagation` set to `slave` mode.

Use the following commands to mount and unmount directories in already running containers:

```
DobbyTool mount com.sky.as.apps_com.bskyb.epgui  /mnt/usb /tmp/newmount ro,bind
DobbyTool unmount com.sky.as.apps_com.bskyb.epgui /tmp/newmount
```

### Type of Change

### Requires Bitbake Recipe changes?
